packages/rlc-common/src/metadata/buildApiExtractorConfig.ts

```diff
@@ -5,15 +5,22 @@ import { Project } from "ts-morph";
 import { RLCModel } from "../interfaces.js";
 
 export function buildApiExtractorConfig(model: RLCModel) {
-  const { packageDetails, isModularLibrary, generateTest, generateSample } =
+  const { packageDetails, isModularLibrary, generateTest } =
     model.options || {};
   const project = new Project();
+
+  let mainEntryPointFilePath = "./dist/esm/index.d.ts";
```

Comment:
I am trying to understand if it's esm, we don't really need to compile the samples and tests? so does the cjs compile result?